pytest-virtualenv/pytest_virtualenv.py

```diff
@@ -16,8 +16,12 @@
 class FixtureConfig(Config):
     __slots__ = ('virtualenv_executable')
 
+# Default values for system resource locations - patch this to change defaults
+DEFAULT_VIRTUALENV_FIXTURE_EXECUTABLE = (cmdline.which('virtualenv') + ['virtualenv'])[0]
+DEFAULT_VIRTUALENV_FIXTURE_EXECUTABLE = '/apps/research/python/bin/virtualenv'
```

Comment:
This looks like a copy-paste error. The second line is clobbering the first, and the second line is AHL-specific.